doc/nrf/known_issues.rst

```diff
@@ -845,14 +851,51 @@ KRKNWK-9214: Pigweed submodule may not be accessible from some regions
 HomeKit
 =======
 
+.. rst-class:: v2-0-0
+
+KRKNWK-14130: Bluetooth LE TX configuration is set to ``0`` dBm by default
+  Appropriate TX power configuration for Bluetooth LE and Thread is required.
+  Requested TX power relates to power on antenna.
+  In case of our multiprotocol samples it is powered after FEM.
+  Initial default FEM gain is set to ``10`` dBm for there to be ``0`` at the antenna output, on the SoC it is ``-10`` dBm.
+  This results in a weak signal on the SoC itself.
+
+  **Workaround:** The appropriate dBm values for Bluetooth LE and Thread need to be configured in the source code.
```

Comment:
@annwoj , @mariuszpos , this whole issue is unclear to me. Can you help me with rephrasing?

- What is the issue? TX conf being set to 0?
- What is the meaning of "our multiprotocol samples"?
- Where should the appropriate values be configured in source code? What are those values?

Reply:
This issue is connected with change that was made in nRF 802.15.4 Radio Driver:
https://github.com/nrfconnect/sdk-nrfxlib/commit/da2ea3770ae223dfa4f13b5851642a790edf06a2#diff-61e4b96979a499a50e373580b526208d2174a7be51f1d16af1266b854251ed52R19
What is the issue? TX conf being set to 0?
- Yes, it should be at least 4 dBm for BLE 
What is the meaning of "our multiprotocol samples"?
- In HomeKit we can build samples as BLE only and as multiprotocol (BLE + Thread)  
Where should the appropriate values be configured in source code? What are those values?
- for BLE protocol should be at least 4 dBm and for multiprotocol at least 8 dBm